functions/src/webhook.ts

```diff
@@ -259,12 +321,17 @@ export class Webhook {
             user,
             targetUsername,
             transferAmount,
-            chatId
+            chatId,
+            userLanguage,
+            groupLanguage
         );
       }
 
       const state = await Database.getInstance().getState(userId);
-      if (!state) return "";
+      if (!state) return Telegram.getInstance().sendText(
+        userId, Language.getString(userLanguage, "errors.not_found"),
+        Language.getKeyboard(userLanguage, this.INLINE_KEYBOARD)
+      );
```

Comment:
This still needs to be addressed. Multi-line returns need curly braces around the if statement.

Reply:
Done!